mmseg/datasets/a2d2.py

```diff
@@ -0,0 +1,44 @@
+import os.path as osp
+
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class A2D2Dataset(CustomDataset):
```

Comment:
We may inherit from `Cityscapes`.

Reply:
@xvjiarui Sorry, but I'm not entirely sure the merit of inheriting from `Cityscapes`.

I think that the A2D2 should be seen as independent from Cityscapes just like the other datasets, despite the option to emulate the same segmentation label categorization.

Could you explain further why you think we should inherit from `Cityscapes`?